lib/rules/declaration-property-value-no-unknown/__tests__/index.js

```diff
@@ -0,0 +1,225 @@
+'use strict';
+
+const { messages, ruleName } = require('..');
+
+testRule({
+	ruleName,
+	config: true,
+
+	accept: [
+		{
+			code: 'a { top: 0; }',
+		},
+		{
+			code: 'a { margin: auto 10em; }',
+		},
+		{
+			code: 'a { margin-inline: calc(100% - 10px); }',
+		},
+		{
+			code: 'a { top: var(--foo); }',
+		},
+		{
+			code: 'a { foo: 1px; }',
+		},
+		{
+			code: 'a { --foo: 1px; }',
+		},
+	],
+
+	reject: [
+		{
+			code: 'a { top: unknown; }',
+			message: messages.rejected('top', 'unknown'),
+			line: 1,
+			column: 10,
+			endLine: 1,
+			endColumn: 17,
+		},
+		{
+			code: 'a { top: red; }',
+			message: messages.rejected('top', 'red'),
+			line: 1,
+			column: 10,
+			endLine: 1,
+			endColumn: 13,
+		},
+		{
+			code: 'a { color: ; }',
+			message: messages.rejected('color', ''),
+			line: 1,
+			column: 11,
+			endLine: 1,
+			endColumn: 12,
+		},
+		{
+			code: 'a { padding: auto; }',
+			message: messages.rejected('padding', 'auto'),
+			line: 1,
+			column: 14,
+			endLine: 1,
+			endColumn: 18,
+		},
+		{
+			code: 'a { display: fixed }',
+			message: messages.rejected('display', 'fixed'),
+			line: 1,
+			column: 14,
+			endLine: 1,
+			endColumn: 19,
+		},
+		{
+			code: 'a { color: rgb(67 67 67 70%); }',
+			message: messages.rejected('color', '70%'),
+			line: 1,
+			column: 25,
+			endLine: 1,
+			endColumn: 28,
+		},
+		{
+			code: 'a { transition: background-color 0.6s ease-in-out 0; }',
+			message: messages.rejected('transition', '0'),
+			line: 1,
+			column: 51,
+			endLine: 1,
+			endColumn: 52,
+		},
+		{
+			code: 'a { margin-top: "10px"; }',
+			message: messages.rejected('margin-top', '"10px"'),
+			line: 1,
+			column: 17,
+			endLine: 1,
+			endColumn: 23,
+		},
+		{
+			code: 'a { color: greem; }',
```

Comment:
suggestion

As false negatives you could add `color: light-grey;` and `color: light-gray;`.
i.e. both are valid